lib/src/components/app_bar/app_bar.dart

```diff
@@ -0,0 +1,435 @@
+import 'package:flutter/foundation.dart';
```

Comment:
Same as in other PR - this can be deleted, must have been duplicated during merge.